src/dataflow/src/server.rs

```diff
@@ -93,12 +93,11 @@ pub enum SequencedCommand {
         /// The identifier of the peek request to cancel.
         conn_id: u32,
     },
-    /// Insert `updates` into the local input named `id`.
+    /// Insert `updates` into local inputs.
     Insert {
         /// Identifier of the local input.
-        id: GlobalId,
-        /// A list of updates to be introduced to the input.
-        updates: Vec<Update>,
+        /// Lists of updates to be introduced to the specified inputs.
+        updates: HashMap<GlobalId, Vec<Update>>,
```

Comment:
nikhil explained offline that this is a HashMap so we can easily construct a session only once per id, but IMO a `Vec<(GlobaId, Vec<Update>)>` is a more natural interface (and makes ruchir's concern above a non-issue). there might be a small performance difference between them (the SequencedCommand::Insert would then either turn the vec into a HashMap or use a HashMap for sessions), but i'd cross that bridge when we come to it. don't feel strongly though